src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -150,19 +215,21 @@ const useGroupInfo = (channelId = null) => {
   const launchGallery = async () => {
     const {success} = await requestExternalStoragePermission();
     if (success) {
-      launchImageLibrary(
-        {
+      ImagePicker.openPicker({
+        mediaType: 'photo',
+        sortOrder: 'asc',
+        smartAlbums: ['RecentlyAdded', 'UserLibrary']
+      }).then(async (imageRes) => {
+        const imageCropped = await ImagePicker.openCropper({
           mediaType: 'photo',
-          maxHeight: 500,
-          maxWidth: 500,
-          includeBase64: true
-        },
-        (res) => {
-          if (!res.didCancel) {
-            uploadImage(res?.assets?.[0]?.uri);
-          }
-        }
-      );
+          path: imageRes.path,
+          width: imageRes.width,
+          height: imageRes.height,
+          cropperChooseText: 'Next',
+          freeStyleCropEnabled: true
+        });
+        uploadImage(imageCropped.path);
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `launchGallery` function has been updated to use `react-native-image-crop-picker`. This allows users to crop the selected image before uploading. Ensure that necessary permissions are handled correctly as this library requires both camera and external storage access permissions.
```diff
+ const launchGallery = async () => {
+   // ... rest of the code
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->